### ISSUE TITLE
Any Keras/TF2.0 implementation?

### ISSUE THREAD

**Issue by maghsk** [cleardusk/3DDFA#152]
Does 3DDFA have any Keras or TensorFlow 2.0 implementation?

**Comment by cleardusk** [comment]
Nope so far. But it may not be hard to re-implement it by Keras or TF2.0, and welcome for any contributions.

**Comment by ammarali32** [comment]
I am ready for this contribution just please provide undamaged archive model

**Comment by cleardusk** [comment]
What does `undamaged archive model` mean?

**Comment by ammarali32** [comment]
I am not able to open or unzip your model on Windows 10 or Linux. trying multiple apps it didn't work

**Comment by cleardusk** [comment]
The default file type of the model trained by PyTorch is `.pth.tar`, there is no need to untar it in PyTorch.

**Comment by ammarali32** [comment]
Yes but i can't work with that file when i try to export onnx model from it still face problems so if you could provide onnx model with a defined batch size not dynamic or  pb model or pt model it will fantastic
